beacon_node/lighthouse_network/src/config.rs

```diff
@@ -305,12 +305,12 @@ impl Default for Config {
         let discv5_config = discv5::ConfigBuilder::new(discv5_listen_config)
             .enable_packet_filter()
             .session_cache_capacity(5000)
-            .request_timeout(Duration::from_secs(1))
+            .request_timeout(Duration::from_secs(2))
             .query_peer_timeout(Duration::from_secs(2))
             .query_timeout(Duration::from_secs(30))
             .request_retries(1)
             .enr_peer_update_min(10)
-            .query_parallelism(5)
+            .query_parallelism(8)
```

Comment:
Why is this bumped?

Reply:
Some simulations or experiments were saying discovery can run faster just by increasing the parallelism. 

I was looking through the configs, and while I was there, figured I'd increase. Happy to leave it as is, if its a concern. 

Its just that I've been meaning to mess with it for a while, so figure in the update, might as well do it here.